code/game/antagonist/station/revolutionary.dm

```diff
@@ -7,7 +7,7 @@ var/datum/antagonist/revolutionary/revs
 	bantype = "revolutionary"
 	feedback_tag = "rev_objective"
 	antag_indicator = "contenderhead"
-	welcome_text = "You are a subversive seeking to demolish the current order on the station by whatever means possible. Recruit friends and strangers alike to bring Nanotrasen's tyranny to an end! Or whatever your objective is."
+	welcome_text = "You are a subversive seeking to demolish the current order on the station by whatever means possible. Recruit friends and strangers alike to bring the SCC's tyranny to an end! Or whatever your objective is."
```

Comment:
Bunch of the `welcome_text` ,  `extended_round_description` , `round_description`  can be moved to `New` or `Initialize` and use the map vars.